desertbot/modules/automatic/Log.py

```diff
@@ -21,9 +21,9 @@
     'PRIVMSG': lambda m: u'<{0}> {1}'.format(m.User.Name, m.MessageString),
```

Comment:
All of these.

And so on, so forth... I skipped a bunch of repeat ones above this, and there are more below.